src/scripts/content/liveagent.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+// ticket
+togglbutton.render(
+  '.ConversationDetailsView:not(.toggl)',
+  { observe: true },
+  function (
+    elem
+  ) {
+    const descriptionSelector = () => {
+      const ticketCode = $('.CodeLabel').textContent.trim();
+      const ticketSubject = $('.ConversationHeaderSubject').textContent.trim();
```

Comment:
We usually try to be more safe in such cases. In case element is not accessible `.textContent` will throw. Can we please fix it? 🙏